tasks/post_to_catalog/test/unit_tests/test_post_to_catalog.py

```diff
@@ -83,6 +83,7 @@ def test_send_record_to_database_happy_path(
             "version": uuid.uuid4().__str__(),
         }
         granule = {
+            "providerId": provider["providerId"],
```

Comment:
Depending on what we decide with the input schema, this may need to be removed.

Reply:
removed